src/theme/css/general.css

```diff
@@ -267,7 +269,7 @@ body {
 	width: 100%;
 }
 #page-wrapper {
-	width: 100%;
+	width: calc(100% - 20px);
```

Comment:
Do you think we could obtain this `20px` from somewhere, to avoid it being hardcoded?

Reply:
Good point. I've extracted a variable for the padding of content inside of `#page-wrapper` to make it clear where the 20 pixels are coming from.